proto/opamp.proto

```diff
@@ -91,6 +91,10 @@ message AgentToServer {
     // A custom message sent from an Agent to the Server.
     // Status: [Development]
     CustomMessage custom_message = 13;
+
+    // A message indicating the components that are available for configuration on the agent.
+    // Status: [Development]
+    AvailableComponents available_components = 14;
```

Comment:
This should have a corresponding capability, right?

Reply:
Yes, it definitely should. Added a capability for ReportsAvailableComponents